src/systemcmds/ver/ver.c

```diff
@@ -179,6 +179,19 @@ int ver_main(int argc, char *argv[])
 					if (git_branch && git_branch[0]) {
 						printf("FW git-branch: %s\n", git_branch);
 					}
+
+					const char *git_tag = px4_firmware_git_tag_string();
+
+					if (git_tag && git_tag[0]) {
+						printf("FW git tag: %s\n", git_tag);
+					}
+
+					const char *ecl_git_hash = px4_ecl_lib_version_string();
+
+					if (ecl_git_hash && ecl_git_hash[0]) {
+						printf("ECL git hash: %s\n", ecl_git_hash);
```

Comment:
This is kind of silly, but it keeps the naming consistent with the others.

```suggestion
						printf("ECL git-hash: %s\n", ecl_git_hash);
```